docs/development.md

```diff
@@ -0,0 +1,108 @@
+# Development
+## Running locally
```

Comment:
New line between the headings? My editor sometimes complains at me for poorly formatted markdown if I don't...